modules/identity/src/Volo.Abp.Identity.EntityFrameworkCore/Volo/Abp/Identity/EntityFrameworkCore/EfCoreIdentityUserRepository.cs

```diff
@@ -136,8 +136,15 @@ select ouRole.RoleId
             int skipCount = 0,
             string filter = null,
             bool includeDetails = false,
+            Guid? roleId = null,
+            Guid? organizationUnitId = null,
             CancellationToken cancellationToken = default)
         {
+            if(roleId.HasValue || organizationUnitId.HasValue)
+            {
+                includeDetails = true;
+            }
+
```

Comment:
Maybe this is not necessary.

Reply:
It's necessary if roleId or organizationUnitId is specified.

Reply:
The `includeDetails` control` sub-collections` load or not, I don't know why it relates to `roleId or organizationUnitId`?

Reply:
Because if `roleId` specified, I need to reach the [Roles collection](https://github.com/abpframework/abp/blob/dev/modules/identity/src/Volo.Abp.Identity.Domain/Volo/Abp/Identity/IdentityUser.cs#L115) for filtering. (Same is valid for `organizationUnitId`)